crates/services/upgradable-executor/src/instance.rs

```diff
@@ -53,6 +56,17 @@ use wasmtime::{
 trait CallerHelper {
     /// Writes the encoded data to the memory at the provided pointer.
     fn write(&mut self, ptr: u32, encoded: &[u8]) -> anyhow::Result<()>;
+
+    /// Peeks the next transactions from the source and returns the size of the encoded transactions.
+    fn peek_next_txs_size<Source>(
```

Comment:
Oh. Can we specify that this is `size_in_bytes` or maybe just `peek_next_txs_bytes`? It wasn't obvious that this was bytes and not # of txs.

I guess within the wasm context it would kinda have to be bytes... but it wasn't obvious immediately to me.